Support using multiple images sources in our CLI (#142)
jddarby/azure-cli-extensions: src/aosm/HISTORY.rst

===============

Unreleased
++++++++

> * Added improvements in how the CNF image sources are handles. Users are now able to specify multiple image sources from all types of registries (not just ACRs).

Reviewer: ```suggestion
* Added: Users can specify multiple image sources from all types of registries (not just ACRs). General improvements in how CNF image sources are handled. 
```

> * Fixed an issue where the namespace would appear twice in the `artifacts.json` file, leading to errors in the publish step of the CLI. 

Reviewer: ```suggestion
* Fixed: Namespace appeared twice in the `artifacts.json` file, leading to errors in the publish step of the CLI. 
```
